cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
I think it'd be fine to expose `updateCacheRepo` as a public API (I only didn't before because it didn't need to be -- but clearly does here)

What's the downside to just printing the exception on the command line, though? Exceptions should be exceptional, and when they do happen they give a lot of useful info for reporting the bug in the GitHub issue.